src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Page.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    [ExcludeFromCodeCoverage]
+    [SuppressMessage("Style", "IDE0016:Use 'throw' expression")]
+    partial class Page
```

Comment:
@Srivatsn, do you want me to ahead and remove these from the PR?

What about the other empty <Rule>.cs files?